hil-test/tests/spi_full_duplex_dma_write_read.rs

```diff
@@ -0,0 +1,108 @@
+//! SPI Full Duplex DMA write + read Test
+//! See issue #2059
+//!
+//! Folowing pins are used:
+//! SCLK    GPIO0
+//! MISO    GPIO2 / GPIO9  (esp32s2 / esp32s3) / GPIO26 (esp32)
```

Comment:
I asked Juraj not to introduce this weird separate grouping for esp32, and I'll ask you, too.

Reply:
Yes. I would like to avoid multiplying bad patterns by copy-paste, please.

Reply:
but seriously does listing the pins in each test make any sense anymore? I don't think so (latest when someone needs to change the pins in lib.rs)

Reply:
@JurajSadel thinks they do, I don't really agree. And updating them will be a nightmare.

Reply:
I will remove them from test's description.